### PR TITLE
archived released bundles after bundle.yaml were removed from charmhub

### PR DESCRIPTION
**Pull request by addyess**
Here's the releases folder
A new release can be created (or an old can be updated) by running 
`./release 1.23 cs:charmed-kubernetes-xxx` from this directory

for charmhub releases
`./release 1.24 ch:charmed-kubernetes`

```
drwxrwxr-x  3 addyess addyess   7 Feb 10 10:15 1.14
drwxrwxr-x  3 addyess addyess   7 Feb 10 10:14 1.15
drwxrwxr-x  3 addyess addyess   7 Feb 10 10:14 1.16
drwxrwxr-x  3 addyess addyess   7 Feb 10 10:14 1.17
drwxrwxr-x  3 addyess addyess   7 Feb 10 10:14 1.18
drwxrwxr-x  2 addyess addyess   6 Feb 10 10:14 1.19
drwxrwxr-x  2 addyess addyess   6 Feb 10 10:14 1.20
drwxrwxr-x  2 addyess addyess   6 Feb 10 10:14 1.21
drwxrwxr-x  2 addyess addyess   6 Feb 10 10:14 1.22
drwxrwxr-x  2 addyess addyess   6 Feb 10 10:13 1.23
lrwxrwxrwx  1 addyess addyess   5 Feb 10 10:15 cs:charmed-kubernetes-124 -> 1.14/
lrwxrwxrwx  1 addyess addyess   5 Feb 10 10:14 cs:charmed-kubernetes-209 -> 1.15/
lrwxrwxrwx  1 addyess addyess   5 Feb 10 10:14 cs:charmed-kubernetes-316 -> 1.16/
lrwxrwxrwx  1 addyess addyess   5 Feb 10 10:14 cs:charmed-kubernetes-410 -> 1.17/
lrwxrwxrwx  1 addyess addyess   5 Feb 10 10:14 cs:charmed-kubernetes-485 -> 1.18/
lrwxrwxrwx  1 addyess addyess   5 Feb 10 10:14 cs:charmed-kubernetes-545 -> 1.19/
lrwxrwxrwx  1 addyess addyess   5 Feb 10 10:14 cs:charmed-kubernetes-596 -> 1.20/
lrwxrwxrwx  1 addyess addyess   5 Feb 10 10:14 cs:charmed-kubernetes-733 -> 1.21/
lrwxrwxrwx  1 addyess addyess   5 Feb 10 10:14 cs:charmed-kubernetes-814 -> 1.22/
lrwxrwxrwx  1 addyess addyess   5 Feb 10 10:13 cs:charmed-kubernetes-862 -> 1.23/
-rwxrwxr-x  1 addyess addyess 663 Feb 10 10:17 release.sh
```

### CODE REVIEW

**Comment by addyess** [comment]
Raw links to the bundle files would be here:
```
https://raw.githubusercontent.com/charmed-kubernetes/bundle/master/releases/1.20/bundle.yaml
```

File: releases/1.23/getstarted.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+# Charmed Distribution of Kubernetes
```
Comment (evilnick) [inline releases/1.23/getstarted.md:2]: Where did this file get sourced from? There is a `{details}` thing that makes it look like a template from somewhere. Anyhow, do we really need it?

File: releases/1.23/post-deployment.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (evilnick) [inline releases/1.23/post-deployment.sh:1]: not sure the purpose of this

File: releases/1.18/tests/amulet_utils.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
+import subprocess
```
Comment (evilnick) [inline releases/1.18/tests/amulet_utils.py:1]: don't think we need this?

File: releases/1.18/post-deployment.sh [hunk 4]
```diff
@@ -0,0 +1,2 @@
+
+k8s-postdeploy
```
Comment (evilnick) [inline releases/1.18/post-deployment.sh:2]: not sure the purpose of this

**Review by evilnick — APPROVED** [review]

**Pull request merged**